phaseshifts/lib/EEASiSSS/EEASiSSS.py

```
@@ -67,7 +67,11 @@
 
 
 def eeasisss(input_file="inputX"):
-    """Wrapper function to call EEASiSSS Fortran library using ctypes"""
+    """Call the EEASiSSS Fortran library using ctypes."""
+    if not os.path.isfile(input_file):
+        raise FileNotFoundError(
+            "Input file '%s' not found" % input_file
```

Ruff (0.14.10)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)